src/core/pd_repo.js

```diff
@@ -0,0 +1,83 @@
+import mongoose from "mongoose";
+
+import data_chunks from "../constants/pd_constants.js";
+
+import logger from "./pd_logger.js";
+
+class repo {
+  constructor() {
+    this.connectionString = [
+      "mongodb+srv://",
+      data_chunks.DB_user,
+      ":",
+      data_chunks.DB_pass,
+      "@cluster0.br6n1f3.mongodb.net/",
+    ].join("");
+    this.connect();
+  }
+  connect = async () => {
```

Comment:
move this connection to config file and get it from there

Reply:
done.